google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -76,6 +77,11 @@
    */
   protected abstract String formatForTest(String resourceName);
 
+  /**
+   * Creates a comparison expression for logging filter.
+   */
+  protected abstract <V> String createComparisonExpression(String name, String op, V value);
```

Comment:
This isn't something that needs to be overridden, so just make a static utility method.